ocs_ci/ocs/ui/pvc_ui.py

```diff
@@ -139,7 +161,18 @@ def pvc_resize_ui(self, project_name, pvc_name, new_size):
         logger.info(f"Search and Select test project {project_name}")
         self.do_click(self.pvc_loc["pvc_project_selector"])
         self.do_send_keys(self.pvc_loc["search-project"], text=project_name)
-        self.do_click(format_locator(self.pvc_loc["test-project-link"], project_name))
+        wait_for_project = self.wait_until_expected_text_is_found(
```

Comment:
@am-agrawa lines 164 to 175 is repeated multiple times. could you please modularize this?

Reply:
Done